bench/workgen/workgen.cpp

```diff
@@ -366,6 +410,25 @@ WorkloadRunner::start_tables_create(WT_CONNECTION *conn)
         // Initially we start creating tables if the database size is less than the create target.
         creating = db_size < _workload->options.create_target;
     }
+
+    std::string uri;
+    std::string mirror_uri = std::string();
+    int creates, retries, status;
+    char rand_chars[DYNAMIC_TABLE_LEN];
+
+    // Add app_metadata to the config to indicate the table was created dynamically (can be
+    // selected for random deletion), the name of the table's mirror if mirroring is enabled,
+    // and if this table is a base table or a mirror. We want these settings to persist over
+    // restarts.
+    std::string base_config =
+      "key_format=S,value_format=S,app_metadata=\"" + DYN_TABLE_APP_METADATA;
+    if (_workload->options.mirror_tables) {
+        base_config += "," + MIRROR_TABLE_APP_METADATA;
```

Comment:
Is there a + "true" missing here for the mirror table app metadata, or is the intention to fill in the mirror table name here?

Reply:
The intention is to fill in the name of the mirror here.